app/src/main/java/com/hedvig/app/feature/chat/data/UserRepository.kt

```diff
@@ -10,8 +10,8 @@ import com.hedvig.app.util.apollo.toFlow
 
 class UserRepository(private val apolloClientWrapper: ApolloClientWrapper) {
 
-    suspend fun newUserSession() =
-        apolloClientWrapper.apolloClient.mutate(NewSessionMutation()).toDeferred().await()
+    fun newUserSession() =
```

Comment:
I think these should be suffixed with `Async()` in accordance with kotlin naming customs